tests/system_tests/conftest.py

```diff
@@ -10,7 +10,7 @@
 import urllib.parse
 from collections.abc import Sequence
 from contextlib import contextmanager
-from typing import Any, Dict, Iterator, List, Optional, Union
+from typing import Any, Dict, Generator, Iterator, List, Optional, Union
```

Comment:
not required, but what do you think of switching to the future annotation typing if you already mucking with this file.

Reply:
oops, I thought I already did that, but that was just `tests/conftest.py`! I can send out a separate PR for this one :)